config-templates/ibm/ibm-system-storage-block-csi-driver/1.11.2/deployment.yaml

```diff
@@ -0,0 +1,122 @@
+apiVersion: v1
+kind: List
+metadata:
+  name: ibm-csi-block
+  namespace: kube-system
+  annotations:
+    version: 1.11.1
```

Comment:
This is as per template version. Please change it to `1.11.2`

Reply:
done